### DISCUSSION TITLE
What are Blender's and its communities thoughts on the Malt renderer?

### DISCUSSION THREAD

**Discussion by eballeste** [bnpr/Malt#386] [General]
Was curious if Blender, its dev's, and the community in general were aware of the efforts behind Malt and what type of feedback or support have they given around its development, if any?

**Comment by dibli-goost** [comment]
There was a discussion about it on the Grease Pencil Module Blender.Chat back in November of last year, I don't know how to link that discussion, but it did happen:


![image](https://user-images.githubusercontent.com/31767887/186996841-a2737024-89bd-40b7-83d8-e6b00a0e6cf1.png)

![image](https://user-images.githubusercontent.com/31767887/186996791-f729f698-d011-401c-81f9-03155ac712a9.png)

Here are some statements from Light regarding it at the time:

![image](https://user-images.githubusercontent.com/31767887/186997004-e6ffe45f-36ca-4d18-8c1d-0c9a7f955091.png)

I don't know if any formal talks happened between Blender Devs and the Malt/BEER team following this, I can't find anything regarding it on the BNPR discord.

**Reply by pragma37** [reply]
I've been in contact with Sebastian and other Blender devs since then.
There's nothing set in stone yet, but I hope to have good news soon. 🤞

**Reply by pragma37** [reply]
> I don't know if any formal talks happened between Blender Devs and the Malt/BEER team following this

BTW, I keep hearing about this mythical Malt/BEER team, but the reality is that (aside from the recent @Kolupsy incorporation 🙌) I have not had any help with Malt neither in the development nor funding side (except for my Patreon and some features funded by users) since 2020.

I think the confusion comes from the BNPR discord (?), where there are many users in the BEER devs group.

**Reply by Kolupsy** [reply]
Probably the 'Beer Dev' roles on the discord server, yeah. I mean I had that role before I contributed anything to Malt OR Beer and I think most of the discord users with that role have made minor contributions before? But I dont really know what the others did to get that role. I guess the 'BEER team' would be miguel and light and the general Beer-Malt confusion does the rest xD

**Reply by dibli-goost** [reply]
> BTW, I keep hearing about this mythical Malt/BEER team, but the reality is that (aside from the recent @Kolupsy incorporation 🙌) I have not had any help with Malt neither in the development nor funding side (except for my Patreon and some features funded by users) since 2020.

Yeah, I've maybe been a bit too hasty about the usage. It seemed like a good colloquial term for referring to "everyone involved with the two projects".

And they are just that too, two projects. Linked, but not the same. I've seen many people refer to them as one in the same, but that's just not really the case is it?

**Reply by pragma37** [reply]
> I've seen many people refer to them as one in the same, but that's just not really the case is it?

At first, Malt was just supposed to be a framework for developers to implement custom render engines, so then someone else could work with Light on implementing BEER on top of it.
https://blendernpr.org/beer-milestones-funding/

I opened my Patreon after the initial development (and funding) ended and kept improving Malt.  
But since BEER development didn't start and most users didn't know how to code, almost everybody could only use the example shaders.

So at that point, it was either wait for BEER to happen and hope Malt didn't rot in the meanwhile, or roll my own solution.
So I started to work on the node system and Malt has been moving towards being a full render engine aimed at end-users since then.
